src/api/actions/startBulkUpload.ts

```diff
@@ -0,0 +1,21 @@
+import { Maybe } from "../../utilities";
+import { UrlToJobId } from "../../types";
+import { makeApiCall } from "./helpers";
+
+export const startBulkUpload = async (
+  apiUri: string,
+  urls: string[]
+): Promise<UrlToJobId[]> => {
+  let response: Maybe<UrlToJobId[]>;
+  try {
+    response = await makeApiCall<UrlToJobId[]>(
+      `${apiUri}/startBulk`,
```

Comment:
This doesn't look like the correct endpoint

Reply:
fixed this